ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthProtocolManager.java

```diff
@@ -210,6 +211,14 @@ private List<Capability> calculateCapabilities(
     capabilities.add(EthProtocol.ETH65);
     capabilities.add(EthProtocol.ETH66);
 
+    // Version 67 removes the GetNodeData and NodeData
+    // Fast sync depends on GetNodeData and NodeData
+    // Do not add eth/67 if fast sync is enabled
+    // see https://eips.ethereum.org/EIPS/eip-4938
+    if (!Objects.equals(SyncMode.FAST, synchronizerConfiguration.getSyncMode())) {
```

Comment:
line 206 checks sync mode for fullSync but it does it differently - would be nice if these two matched. idk which is better

Reply:
Yes, created some nodes with fast sync: https://github.com/hyperledger/besu/issues/4596